config/core_pools.json

```diff
@@ -52,23 +54,21 @@
     "0xade4a71bb62bec25154cfc7e6ff49a513b491e81000000000000000000000497": "rETH-WETH-BPT",
     "0xb61371ab661b1acec81c699854d2f911070c059e000000000000000000000516": "ezETH/wstETH",
     "0xc2598280bfea1fe18dfcabd21c7165c40c6859d30000000000000000000004f3": "wstETH/sfrxETH",
+    "0xd0ec47c54ca5e20aaae4616c25c825c7f48d40690000000000000000000004ef": "rETH/wETH BPT",
     "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH",
     "0xa1a8bf131571a2139feb79401aa4a2e9482df6270002000000000000000004b4": "wstETH:4pool",
     "0x4b3af34eb1135d59df8b9cdc2ff07d30d05334c400000000000000000000050d": "weETH/rETH",
     "0xca8ecd05a289b1fbc2e0eaec07360c4bfec07b6100020000000000000000051d": "2CLP-AUSDC-USDC",
     "0x14abd18d1fa335e9f630a658a2799b33208763fa00020000000000000000051f": "2CLP-AUSDT-USDT",
     "0x125bc5a031b2db6733bfa35d914ffa428095978b000200000000000000000514": "ECLP-AUSDC-AUSDT",
-    "0xd0ec47c54ca5e20aaae4616c25c825c7f48d40690000000000000000000004ef": "rETH/wETH BPT",
     "0x423a1323c871abc9d89eb06855bf5347048fc4a5000000000000000000000496": "4POOL",
     "0x32df62dc3aed2cd6224193052ce665dc181658410002000000000000000003bd": "RDNT-WETH"
   },
-  "optimism": {},
```

Comment:
I think this needs to be in there as an empty map or some things fail on lookup, because we added op to a list somewhere that gets read.

Reply:
c98c7fb